### DISCUSSION TITLE
Welcome to Glicol Discussions!

### DISCUSSION THREAD

**Discussion by chaosprint** [chaosprint/glicol#34] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by jonnyhotchkiss** [comment]
Hi! Just found your reddit, then this, where to start?!

Hoping to understand enough to use and hopefully even contribute to the development of all these awesome audio|web toys and tools, and even collaborate and integrate, in development and support and even performing and integrating... **L**et's **E**mploy **G**lobal **O**bjects!

I guess my first questions are what is Glicol, how does it work, what can it do? Looking for those answers now!

Hi! I am here to learn, assist, and discover

**Reply by chaosprint** [reply]
The best place to start is the quick tour here: https://glicol.org/tour
feel free to ask more if you have any questions on the tutorial
